[ADD] accounting/l10n: new localization page for United Arab Emirates (#2510)
odoo/documentation: content/applications/finance/fiscal_localizations/united_arab_emirates.rst

     - Payable
   * - 101004
     - Bank
     - Bank and Cash
   * - 105001
     - Cash
     - Bank and Cash
   * - 100001
     - Liquidity Transfer
     - Current Assets
   * - 101002
     - Outstanding Receipts
     - Current Assets
   * - 101003
     - Outstanding Payments
     - Current Assets
   * - 104041
     - VAT Input
     - Current Assets
   * - 100103
     - VAT Receivable
     - Non-current Assets
   * - 101001
     - Bank Suspense Account
     - Current Liabilities
   * - 201017
     - VAT Output
     - Current Liabilities
   * - 202001
     - End of Service Provision
     - Current Liabilities
   * - 202003
     - VAT Payable
     - Non-current Liabilities
   * - 999999
     - Undistributed Profits/Losses
     - Current Year Earnings
   * - 400003
     - Basic Salary
     - Expenses
   * - 400004
     - Housing Allowance
     - Expenses
   * - 400005
     - Transportation Allowance
     - Expenses
   * - 400008
     - End of Service Indemnity
     - Expenses

Taxes
=====

To access your taxes, go to :menuselection:`Accounting --> Configuration --> Taxes`.
Activate/deactivate, or :doc:`configure </applications/finance/accounting/taxation/taxes/taxes>` the

> taxes relevant to your business by clicking on them. Remember to set only tax accounts on the **5%**

Reviewer: Remember to set only --> remember to only set
(I know I commented on this part in my initial review, I apologize if I got it wrong the first time)

tax group, as other groups do not need closing. To do so, enable the :doc:`developer mode
<../../general/developer_mode>` and go to :menuselection:`Configuration --> Tax Groups`. Then, set a
:guilabel:`Tax current account (payable)`, :guilabel:`Tax current account (receivable)`, and an

> :guilabel:`Advance Tax payment account` for the **5%** group.

Reviewer: Yes, that's great, thanks!

field and select one of the services.

.. _uae/payroll:

Payroll
=======

The :guilabel:`U.A.E. - Payroll` module includes all the rules configured under the
:guilabel:`U.A.E. Employee Payroll Structure` in the **Salary Rules** section as per the U.A.E.
rules and regulations. These rules are linked to the corresponding accounts in the **Chart of
Accounts**.

.. image:: united_arab_emirates/uae-localization-salary-rules.png
   :align: center
   :alt: The U.A.E. Employee Payroll Structure.

Salary rules
------------

To apply these rules to an employee's contract, go to :menuselection:`Payroll --> Contracts -->
Contracts` and select the employee's contract. In the :guilabel:`Salary Structure Type` field,
select :guilabel:`UAE Employee`.

.. image:: united_arab_emirates/uae-localization-salary-structure.png
   :align: center
   :alt: Select the Salary Structure Type to apply to the contract.

Under the :guilabel:`Salary Information` tab, you can find details such as the:

- :guilabel:`Wage`;
- :guilabel:`Housing Allowance`;
- :guilabel:`Transportation Allowance`;
- :guilabel:`Other Allowances`;

> - :guilabel:`Number of Days`: used to calculate :ref:`End of Service Provision

Reviewer: used to calculate the End of service provision

  <ae-end-of-service-provision>`.

.. note::

>    **Leave deduction** is calculated using a salary rule linked to the **Unpaid leave** time-off

Reviewer: The **Leave deduction** ?

   type; however, any other deductions or reimbursements are made *manually* using other inputs.
   Additionally, **overtime** is added *manually* through :menuselection:`Work Entries --> Work
   Entries`, and **salary attachments** are generated from :menuselection:`Contracts --> Salary
   Attachments`. :guilabel:`Create` an attachment and select the :guilabel:`Employee` and the
   :guilabel:`Type (Attachment of Salary, Assignment of Salary, Child Support)`.

.. tip::

>    If you wish to prevent a rule appearing on a paycheck, go to :menuselection:`Payroll -->

Reviewer: prevent a rule from appearing

   Configuration --> Structures`. Click on the **U.A.E. structure**, select the rule to hide, and
   uncheck :guilabel:`Appears on Payslip`.

> .. _ae-end-of-service-provision:

Reviewer 2: ```suggestion
.. _uae-end-of-service-provision:
```
I assume the ref is missing a "u"

> End of Service Provision

Reviewer 2: ```suggestion
End of service provision
```

------------------------

The provision is defined as the total monthly allowance *divided* by 30 and then *multiplied* by the
number of days set in the field :guilabel:`Number of days` at the bottom of a contract's form.

The provision is then calculated via a salary rule associated with two accounts: the **End Of
Service Indemnity (Expense account)** and the **End of Service Provision (Non-current Liabilities

> account)**. The latter is used to pay off the **End of service** amount by settling it with the
> **Payables account**.

Reviewer 2: ```suggestion
account)**. The latter is used to pay off the **end of service amount** by settling it with the
**payables account**.
```

.. note::

>    The **End of service** amount is calculated based on the gross salary and the employee’s contract

Reviewer: employee's contract's start and end dates
(or the start and end dates of the employee's contract)

>    The **End of service** amount is calculated based on the gross salary and the start and end dates

Reviewer 2: ```suggestion
   The end of service amount is calculated based on the gross salary and the start and end dates
```

   of the employee’s contract.

Invoices
--------

> The U.A.E localization package allows to generate invoices in English, Arabic, or both. The

Reviewer 2: ```suggestion
The U.A.E localization package allows the generation of invoices in English, Arabic, or both. The
```

.. list-table::
   :header-rows: 1

   * - Name
     - Technical name
     - Description
   * - :guilabel:`United Arab Emirates - Accounting`
     - ``l10n_ae``
     - Default :doc:`fiscal localization package </applications/finance/fiscal_localizations>`.
       Contains all accounts, taxes, and reports.
   * - :guilabel:`U.A.E. - Payroll`
     - ``l10n_ae_hr_payroll``
     - Contains all rules, calculations, and salary structures.
   * - :guilabel:`U.A.E. - Payroll with Accounting`
     - ``l10n_ae_hr_payroll_account``
     - Contains all rules, calculations, salary structures, and accounts related to the rules.
   * - :guilabel:`United Arab Emirates - Point of Sale`
     - ``l10n_ae_pos``
     - Contains the U.A.E-compliant PoS receipt.

.. image:: united_arab_emirates/l10n-ae-modules.png
   :align: center
   :alt: Select the modules to install.

Chart of accounts
=================

Go to :menuselection:`Accounting --> Configuration --> Chart of Accounts` to view all default
accounts available for your localization package. You can filter by :guilabel:`code` using the
numbers on the far left or by clicking on :menuselection:`Group By --> Account Type`. You can

> **activate/deactivate, or configure** specific accounts from this menu according to your needs.

Reviewer 3: ![deactivate](https://user-images.githubusercontent.com/104567387/233306089-9cbdaea5-203e-48f9-9b7d-5705e599cd7e.png)
Enable/disable in this case means allow/disallow reconciliation, I'll change it

   * - 100001
     - Liquidity Transfer
     - Current Assets
   * - 101002
     - Outstanding Receipts
     - Current Assets
   * - 101003
     - Outstanding Payments
     - Current Assets
   * - 104041
     - VAT Input
     - Current Assets
   * - 100103
     - VAT Receivable
     - Non-current Assets
   * - 101001
     - Bank Suspense Account
     - Current Liabilities
   * - 201017
     - VAT Output
     - Current Liabilities
   * - 202001
     - End of Service Provision
     - Current Liabilities
   * - 202003
     - VAT Payable
     - Non-current Liabilities
   * - 999999
     - Undistributed Profits/Losses
     - Current Year Earnings
   * - 400003
     - Basic Salary
     - Expenses
   * - 400004
     - Housing Allowance
     - Expenses
   * - 400005
     - Transportation Allowance
     - Expenses
   * - 400008
     - End of Service Indemnity
     - Expenses

Taxes
=====

To access your taxes, head to :menuselection:`Accounting --> Configuration --> Taxes`.
Activate/deactivate, or :doc:`configure </applications/finance/accounting/taxation/taxes/taxes>` the
taxes relevant to your business by clicking on them. Remember only to set accounts on the **5%** tax

> group, as other groups do not need closing.

Reviewer 3: I added instructions on setting tax accounts on the tax group. 5% means indeed 5% in their name, I won't specify this

.. tip::

>    Use the :guilabel:`Group By --> Tax Type` filter to group taxes by type and have a clear overview

Reviewer 3: Removed the tip anyways, it was pointless

   of the different taxes.

.. image:: united_arab_emirates/uae-localization-taxes.png
   :align: center
   :alt: Preview of the U.A.E. localization package's taxes.

Tax report
----------

To create a tax report when closing your period, head to :menuselection:`Accounting --> Reporting
--> Tax Report`. The closing period corresponds to the period configured in
:menuselection:`Accounting --> Configuration --> Settings --> Taxes`, but you can always change it
by clicking on the **calendar icon** and selecting the period you wish to close.

Journals
--------

To configure your journals, go to :menuselection:`Accounting --> Configuration --> Journals`. You
can modify the pre-configured journals according to your needs or create new journals with five
possible types: **Sales, Purchase, Bank, Cash**, and **Miscellaneous**.

> 

Reviewer 3: Good point

Currency exchange rates
-----------------------

Currency exchange rates are automatically updated using the U.A.E Central Bank rates, but by default

> the interval is set to *manual*. If you wish to use the automated currency rate update, change the

Reviewer: I would put the word "interval" in guilabel to highlight the fact that it's the name of the field here?

:guilabel:`U.A.E. Employee Payroll Structure` in the **Salary Rules** section as per the U.A.E.
rules and regulations. These rules are linked to their corresponding accounts in the **Chart of
Accounts**.

.. image:: united_arab_emirates/uae-localization-salary-rules.png
   :align: center
   :alt: The U.A.E. Employee Payroll Structure.

Salary rules
------------

To apply these rules to an employee's contract, go to :menuselection:`Payroll --> Contracts -->
Contracts` and select the contract's employee. In the :guilabel:`Salary Structure Type` field,
select :guilabel:`UAE Employee`.

.. image:: united_arab_emirates/uae-localization-salary-structure.png
   :align: center
   :alt: Select the Salary Structure Type to apply to the contract.

Under the :guilabel:`Salary Information` tab, you can find details such as:

- :guilabel:`Wage`;
- :guilabel:`Housing Allowance`;
- :guilabel:`Transportation Allowance`;
- :guilabel:`Other Allowances`;
- :guilabel:`Number of Days`: how the **End of Service Provision** is calculated.

.. note::
   **Leave deduction** is calculated using a salary rule linked to the unpaid leave time-off type;

>    however, any other deductions or reimbursements are done *manually* using other inputs.

Reviewer: are done *manually* --> are made *manually*

   Additionally, **overtime** is added *manually* through :menuselection:`Work Entries --> Work
   Entries`, and **salary attachments** are generated from :menuselection:`Contracts --> Salary
   Attachments`. :guilabel:`Create` an attachment and select the :guilabel:`Employee` and the

>    :guilabel:`Type (Attachment of Salary, Assignment of Salary, Child Support)`.

Reviewer: Okay! :)

.. tip::
   If you wish to avoid a rule to appearing on a paycheck, go to :menuselection:`Payroll -->
   Configuration --> Structures`. Click on the **U.A.E. structure**, select the rule to hide, and
   uncheck :guilabel:`Appears on Payslip`.

End of Service Provision
------------------------

The provision is defined as the total monthly allowance *divided* by 30 and then *multiplied* by the
number of days set in the field :guilabel:`Number of days` at the bottom of a contract's form.

The provision is then calculated via a salary rule associated with two accounts: the **End Of
Service Indemnity (Expense account)** and the **End of Service Provision (Non-current Liabilities
account)**. The latter is used to pay off the **End of service** amount by settling it with the
**Payables account**.

.. note::

>    The **End of service** calculations is based on the gross salary according to the United Arab

Reviewer: I would suggest: The **End of service** amount [or other word more suitable] is calculated based on the employee's gross salary (according to the United Arab Emirates' HR laws), as well as the contract's start and end dates.

(Is the "according to the United Arab Emirates' HR laws" necessary here since we're talking about the UAE localization since the beginning?)

   Emirates' HR laws. The start and end dates of the employee’s contracts trigger the calculations.

Invoices
--------

> Invoices generated with the U.A.E localization installed can be in English, Arabic, or both. The

Reviewer: Invoices generated with the U.A.E localization installed can be in English --> When the U.A.E localization package has been installed, invoices can be generated in English ... 
or: The U.A.E localization package allows you to generate invoices in English ...

(the "can be in English, ..." sounds weird to me)

> localization also includes a line to display the **VAT amount** per line.

Reviewer: Could there be a blank line missing at the end of this file?

Reviewer 3: Not sure what you meant here, there is a blank line present (it's not missing). This is mandatory otherwise the check fails on GitHub.

Reviewer: Yes, that's what I wanted to check  :) 
(I don't see it on Github, I suppose I should download (whatever the correct term is) your branch to see it?)
